src/linagora.esn.unifiedinbox/app/components/inbox-configuration/filter-definition/configuration-filter-definition.controller.js

```diff
@@ -123,13 +128,17 @@ function inboxConfigurationFilterDefinitionController(
     case JMAP_FILTER.CONDITIONS.TO.JMAP_KEY:
     case JMAP_FILTER.CONDITIONS.CC.JMAP_KEY:
     case JMAP_FILTER.CONDITIONS.RECIPIENT.JMAP_KEY:
-      conditionValue = self.newFilter.stakeholders[0].email;
+      conditionValue = !!self.newFilter.stakeholders.length && self.newFilter.stakeholders[0].email || '';
```

Comment:
Why is `!!` used here?

Reply:
JS trick for bool check, here to test array instead of [] > 0 ....

Reply:
Yes I know what `!!` does, but I don't get why using it on length (implicit conversion number to boolean) is needed. Just asking, I approved anyway :)